src/ingest-pipeline/airflow/dags/utils.py

```diff
@@ -845,6 +844,27 @@ def pythonop_get_dataset_state(**kwargs) -> JSONType:
         'local_directory_full_path': full_path,
         'metadata': metadata,
     }
+
+    if ds_rslt['entity_type'] == 'Dataset':
+        http_hook = HttpHook('GET', http_conn_id='entity_api_connection')
+        endpoint = f"datasets/{ds_rslt['uuid']}/organs"
+        try:
+            response = http_hook.run(endpoint,
+                                     headers=headers,
+                                     extra_options={'check_response': False})
+            response.raise_for_status()
+            organs_query_rslt = response.json()
+            print('organs_query_rslt:')
+            pprint(organs_query_rslt)
+        except HTTPError as e:
+            print(f'ERROR: {e}')
+            if e.response.status_code == codes.unauthorized:
+                raise RuntimeError('entity database authorization was rejected?')
+            else:
+                print('benign error')
+                return {}
+        rslt['organs'] = [entry['organ'] for entry in organs_query_rslt]
```

Comment:
might be best to move this inside of the try: statement, since organs_query_rslt is scoped there.

Reply:
Done, though the statement was unreachable if the try: failed.